mscore/scoreview.cpp

```diff
@@ -4686,8 +4686,10 @@ void ScoreView::cmdRepeatSelection()
             return;
             }
       if (!selection.isRange()) {
-            qDebug("wrong selection type");
-            return;
+            ChordRest* cr = _score->getSelectedChordRest();
+            if (!cr)
+                return;
```

Comment:
4 space indent here, but 6 else?

Reply:
Arrgh!  I set up a new build environment and hadn't bothered installing my usual formatting because I knew it was changing, so I was using the QtCreator defaults and fixing them manually.  Missed this one, thanks for catching.  Harmless in the end I guess, but still...

Reply:
fix pushed